libs/portals/core/src/components/PortalNavigation/PortalNavigation.tsx

```diff
@@ -19,25 +20,33 @@ const findActiveNav = (
   return findActiveNav(activeChild) || activeChild
 }
 
-export function PortalNavigation({ navigation }: PortalNavigationProps) {
+export function PortalNavigation({
+  navigation,
+  title = '',
+}: PortalNavigationProps) {
   const { formatMessage } = useLocale()
   const nav = useNavigation(navigation)
   const { lg } = useBreakpoint()
   const activeNav = useMemo(() => findActiveNav(nav), [nav])
-
+  const params = useParams()
   if (!nav) {
     return null
   }
 
   return (
     <Navigation
-      title={formatMessage(nav.name)}
+      title={title ? title : formatMessage(nav.name)}
       baseId={'navigation'}
       isMenuDialog={!lg}
       activeItemTitle={activeNav ? formatMessage(activeNav.name) : undefined}
-      renderLink={(link, item) =>
-        item?.href ? <Link to={item.href}>{link}</Link> : link
-      }
+      renderLink={(link, item) => {
+        Object.keys(params).forEach((key) => {
+          if (item?.href) {
+            item.href = item.href.replace(`:${key}`, params[key] as string)
+          }
+        })
+        return item?.href ? <Link to={item.href}>{link}</Link> : link
+      }}
```

Comment:
Dangerous to mutate argument objects like this. Possibly the `item.href` is only replaced once and then has incorrect url when navigating further.